### DISCUSSION TITLE
What is the proper syntax for looking for campsites with 30 and 50 amp hookups?

### DISCUSSION THREAD

**Discussion by n-7ity** [juftin/camply#98] [Q&A] [answered]
interested beacause of EV charging

**Comment by juftin** [comment, marked as answer]
Hey @n-7ity - I haven't implemented the ability to filter campsites by attributes like this yet. Currently only [Filtering by Equipment Type](https://juftin.com/camply/README.html#searching-for-a-campsite-that-fits-your-equipment) is supported. I hope to have a solution to this someday soon
